File: PERL/prime.pl
#program to print the prime numbers
$count=2;
while($count<100)
{
	$i=2;

	while($i<$count)
	{
		if($count%$i==0)
		{
			$flag=0;
			break;
		}
		else
		{
			$flag=1;
		}
		$i++;
	}
	
	if(flag)
	{
		print("\n$count");
	}
	$count++;
}
		

File: java/prime.java
class prime
{
	static void main(String args[])
	{
		int prime=0;
	 for(int i=2;i<100;++i)
	 {
		for(int j=2;j<i;++j)
		{
			if(i%j!=0)
				prime=1;
			else
			{
				prime=0;
			 	break;
			}
		}
		if(prime==1)
		System.out.println(i);
	}}
}